release-notes/release-notes-mysql-wsrep-8.0.33-26.14.txt

```diff
@@ -0,0 +1,50 @@
+# Changes in MySQL-wsrep 8.0.33-26.14 (2023-06-16)
+
+Codership is pleased to announce a new GA release of Galera Cluster for MySQL
+8.0, consisting of MySQL-wsrep 8.0.33 and wsrep API version 26.
+
+This release incorporates all changes up to MySQL 8.0.33.
+
+Galera Cluster 8.0 is available as targeted packages and package repositories
+for Debian, Ubuntu, CentOS and RHEL. Obtaining packages using a package
+repository removes the need to download individual files and facilitates the
+deployment and upgrade of Galera nodes.
+
+This and future releases will be available from https://www.galeracluster.com.
+
+The source repositories and bug tracking are now on
+https://github.com/codership/mysql-wsrep.
+
+Notable changes and fixes in MySQL-wsrep since the latest release 8.0.32-26.13:
+- when inside a stored function or trigger, the current statement transaction
+should not allow commit or rollback
+- nodes now use the same server UUID (generated on the seed node) to binlog
```

Comment:
This behavior is configurable with the `wsrep_sync_server_uuid` mentioned below. It was decided that the default behavior was left as it was before the change (i.e. all nodes generate their own server uuids). Maybe this item could be merged with "New wsrep_sync_server_uuid` configuration variable".